build.assets/tooling/cmd/difftest/_fixtures/ast/fork-point/a_simple_test.go

```diff
@@ -0,0 +1,16 @@
+package main
```

Comment:
Please call this directory `testdata`, not `_fixtures`. 

`testdata` is a special name understood by the `go` tool, `_fixtures` is not. This is especially important here because the test data is actually Go source code, but we don't want the `go` tool to try and build or test this code.